assets/csidriveroperators/aws-ebs/mgmt_assets/01_operator_role.yaml

```diff
@@ -0,0 +1,86 @@
+# RBAC needed to start the AWS Operator in mgmt cluster
```

Comment:
I like separate assets dirs, we could have multiple copies of assets for now, but in the long run we should have a way how to generate hypershift assets from standalone or the other way around.

Reply:
Yeah I was thinking that - we could use `kustomize` or another tool to DRY up the YAMLs, so as we don't have to make duplicate changes all over. But I would prefer to avoid using golang to mutate these YAMLs unless we really need some runtime data.